ml-agents/mlagents/trainers/demo_loader.py

```diff
@@ -93,20 +93,28 @@ def demo_to_buffer(
     demo_buffer = make_demo_buffer(info_action_pair, behavior_spec, sequence_length)
     if expected_behavior_spec:
         # check action dimensions in demonstration match
-        if behavior_spec.action_shape != expected_behavior_spec.action_shape:
+        if (
+            behavior_spec.action_spec.continuous_action_size
```

Comment:
same comment as above :
```suggestion
            behavior_spec.actions.num_continuous
```